rskj-core/src/main/java/co/rsk/rpc/modules/eth/ProgramRevert.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rpc.modules.eth;
+
+import com.sun.istack.NotNull;
+
+import javax.annotation.Nonnull;
+
+public class ProgramRevert {
+    @Nonnull
+    private final String reason;
+
+    @Nonnull
+    private final byte[] data;
+
+
+    public ProgramRevert(String reason, @Nonnull byte[] data) {
```

Comment:
As not having reason argument it is a common usage it would be nice to have a single constructor like 
`    public ProgramRevert(@Nonnull byte[] data) { this("", data)}
`